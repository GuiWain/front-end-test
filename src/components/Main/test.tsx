import { render } from 'utils/test-utils'

import Main from '.'

describe('<Main />', () => {
  it('should render the Main', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
