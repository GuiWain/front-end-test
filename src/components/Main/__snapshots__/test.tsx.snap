// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the Main 1`] = `
.c0 {
  background: #FFFFFF;
}

<div
  class="c0"
>
  <h1>
    Main
  </h1>
</div>
`;
